File: php/app/SiteManagement/Models/DartEvent.php
<?php

namespace TrentonDarts\SiteManagement\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class DartEvent extends Model
{
    public static $eventTypes = [
        1 => 'GTDL Event',
        2 => 'Charity Dart Event',
        3 => 'Regional Event',
        4 => 'GTDL Sponsored Event',
        5 => 'GTDL All Stars',
        6 => 'GTDL Player Event',
        7 => 'Charity Event',
        8 => 'DPNY Series',
        9 => 'CDC Series',
        10 => 'DPNJ Series',
        11 => 'Qualifier'];

    use SoftDeletes;
    /**
     * The database table used by the model.
     *
     * @var string
     */
    protected $table = 'dart_events';

    /**
     * The attributes that are mass assignable.
     *
     * @var array
     */
    protected $fillable = ['id',
        'name',
        'eventContact',
        'eventContact2',
        'eventTypeId',
        'eventDate',
        'eventEndDate',
        'dartType',
        'imageFileId',
        'posterFileId',
        'posterFile',
        'url',
        'facebookUrl',
        'hostName',
        'hostUrl',
        'hostPhone',
        'locationName',
        'address1',
        'address2',
        'city',
        'state',
        'zip',
        'registrationStartTime',
        'registrationEndTime',
        'dartStart',
        'mapUrl',
        'description'];

    public function eventType()
    {
        return self::$eventTypes[$this->eventTypeId];
    }

    public function results()
    {
        return $this->hasMany('TrentonDarts\SiteManagement\Models\DartEventResult', 'eventId');
    }

    public function getEventDateAttribute($value)
    {
        if($value == null)
            return '';
        return $value;
    }

    public function getEventEndDateAttribute($value)
    {
        if($value == null)
            return '';
        return $value;
        //return date('m-d-Y', strtotime($value));
    }
}

File: src/DartLeague/DartLeague.Repositories/LeagueData/DartEvent.cs
using System;

namespace DartLeague.Repositories.LeagueData
{
    public partial class DartEvent
    {
        public int Id { get; set; }
        public string Address1 { get; set; }
        public string Address2 { get; set; }
        public string City { get; set; }
        public string DartStart { get; set; }
        public string DartType { get; set; }
        public string Description { get; set; }
        public string EventContact { get; set; }
        public string EventContact2 { get; set; }
        public DateTime EventDate { get; set; }
        public DateTime? EventEndDate { get; set; }
        public int EventTypeId { get; set; }
        public string FacebookUrl { get; set; }
        public string HostName { get; set; }
        public string HostPhone { get; set; }
        public string HostUrl { get; set; }
        public int ImageFileId { get; set; }
        public bool IsTitleEvent { get; set; }
        public string LocationName { get; set; }
        public string MapUrl { get; set; }
        public string Name { get; set; }
        public string PosterFile { get; set; }
        public int PosterFileId { get; set; }
        public string RegistrationEndTime { get; set; }
        public string RegistrationStartTime { get; set; }
        public string State { get; set; }
        public string Url { get; set; }
        public string Zip { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime? DeletedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }
}
